jetty-server/pom.xml

```diff
@@ -74,5 +74,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.googlecode.json-simple</groupId>
+      <artifactId>json-simple</artifactId>
```

Comment:
I'd prefer the test used our own JSON parser rather than bring in another dependency

Reply:
I agree, remove this dependency and use `jetty-util-ajax`.